admin: describe traffic analysis with Wireshark

Totktonada opened the pull request:
This recipe uses the Wireshark plugin https://github.com/tarantool/tarantool-dissector created by Mikhail Galanin (@negram).

Totktonada committed:
admin: describe traffic analysis with Wireshark

This recipe uses the Wireshark plugin [1] created by Mikhail Galanin
(@negram).

[1]: https://github.com/tarantool/tarantool-dissector

Totktonada commented:
Friendly built by CI: https://docs.d.tarantool.io/en/doc/inspect-traffic-with-wireshark/book/admin/server_introspection/#inspect-traffic (my gratitude to @NickVolynkin).

![image](https://user-images.githubusercontent.com/92749/177434130-c54d355e-4c86-450f-b37b-ad459d4ea994.png)

I'm not very familiar with processes around the documentation repository. Is something else expected from me at this point?

patiencedaur commented on doc/book/admin/server_introspection.rst:
```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
+
+--------------------------------------------------------------------------------
+Inspect traffic
+--------------------------------------------------------------------------------
```
If possible, please bring *all* header formatting in accordance with our style guide: https://www.tarantool.io/en/doc/latest/contributing/docs/markup/headings/

I. e.
```
Inspect traffic
---------------
```
not
```
---------------------------
Inspect traffic
---------------------------
```

Totktonada replied:
No problem. Added as a separate commit before the main change. I would appreciate if this pull request will go to `latest` branch without squashing to keep the history easy to follow.

patiencedaur replied:
Unfortunately, only "squash and merge" is enabled for this repo.

patiencedaur commented on doc/book/admin/server_introspection.rst:
```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
+`Wireshark plugin <https://github.com/tarantool/tarantool-dissector>`_ to
+simplify analysis of Tarantool's traffic.
+
+Please, consider several simple steps to enable the plugin.
```
```suggestion
Inspecting binary traffic is a boring task. We offer a
`Wireshark plugin <https://github.com/tarantool/tarantool-dissector>`_ to
simplify the analysis of Tarantool's traffic.

To enable the plugin, follow the steps below.
```
Let the user decide just how simple that is. We did our job by describing the process thoroughly.

Totktonada replied:
Sure, it looks better now. Updated.

patiencedaur commented on doc/book/admin/server_introspection.rst:
```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
+(Please, refer to chapter
+`Plugin folders <https://www.wireshark.org/docs/wsug_html_chunked/ChPluginFolders.html>`_
+in the Wireshark documentation for location of the plugin directory on macOS and
+Windows.)
```
```suggestion
(For the location of the plugin directory on macOS and
Windows, please refer to the
`Plugin folders <https://www.wireshark.org/docs/wsug_html_chunked/ChPluginFolders.html>`_ chapter
in the Wireshark documentation.)
```
Here are some syntax-related remarks:
- "Please" with a comma is a desperate "please" (like someone is making the same mistake for far too long). "Please" without a comma is just polite.
- I changed the sentence structure to enhance the [topic, not comment](https://en.wikipedia.org/wiki/Topic_and_comment).

Totktonada replied:
I agree. Updated.

patiencedaur commented on doc/book/admin/server_introspection.rst:
```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
+in the Wireshark documentation for location of the plugin directory on macOS and
+Windows.)
+
+Run Wireshark GUI and ensure that the plugins are loaded:
```
```suggestion
Run the Wireshark GUI and ensure that the plugins are loaded:
```

Totktonada replied:
Updated.

patiencedaur commented on doc/book/admin/server_introspection.rst:
```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
+
+Run Wireshark GUI and ensure that the plugins are loaded:
+
+* Open ``Help`` -> ``About Wireshark`` -> ``Plugins``.
```
```suggestion
* Open :guilabel:`Help` > :guilabel:`About Wireshark` > :guilabel:`Plugins`.
```
Consider using the `:guilabel:` directive to indicate GUI elements, as mentioned in our [style guide](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/gui/).

Totktonada replied:
Missed it, thanks! Updated.

patiencedaur commented on doc/book/admin/server_introspection.rst:
```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
+* Open ``Help`` -> ``About Wireshark`` -> ``Plugins``.
+* Find ``MessagePack.lua`` and ``tarantool.dissector.lua`` in the list.
+
+Now you can inspect Tarantool incoming and outcoming packets with user-friendly
```
```suggestion
Now you can inspect incoming and outgoing Tarantool packets with user-friendly
```

Totktonada replied:
Updated.

Totktonada replied:
Missed outcoming -> outgoing change at first glance, updated it now as well.

patiencedaur reviewed (changes requested):
Thanks for the patch! Here are some language-related comments.

Totktonada committed:
admin: adjust heading markup to adhere our style

I made the change for the 'Server introspection' article, because I'm
going to add a section here in the next commit.

The style is described in
https://www.tarantool.io/en/doc/latest/contributing/docs/markup/headings/

Totktonada commented:
@patiencedaur Thank you for detailed comments! I updated the patchset.

patiencedaur reviewed (approved):
Thanks for the patch!
